src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -191,7 +267,9 @@ const SampleChatInfoScreen = () => {
               <View style={styles.row}>
                 <View style={styles.column}>
                   <View style={styles.containerGroupName}>
-                    <Text style={styles.groupName}>{trimString(channelInfo?.name, 20)}</Text>
+                    <Text numberOfLines={1} style={styles.groupName}>
+                      {getChatName(channelInfo?.name, profile?.myProfile?.username)}
+                    </Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text inside the `Text` component has been changed. Ensure that the new function `getChatName` is working as expected and is properly tested.



<!-- This is an auto-generated comment by OSS CodeRabbit -->